File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex23.java
import java.util.Scanner;

public class Ex23 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite um numero");
        var number = scan.nextDouble();

        if ((int) number == number) {
            System.out.println("Inteiro");
        } else {
            System.out.println("Decimal");
        }
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/c/Ex23.cs
using System;

class Ex23
{
    static void Main(string[] args)
    {
        Console.Write("Digite um numero: ");
        double numero = double.Parse(Console.ReadLine());

        string result = numero == Math.Round(numero) ? "Inteiro" : "Decimal";

        Console.WriteLine(result);
    }
}